pkg/i18n/english.go

```diff
@@ -1106,6 +1107,7 @@ func EnglishTranslationSet() TranslationSet {
 		SetAsUpstream:                       "Set as upstream of checked-out branch",
 		SetUpstream:                         "Set upstream of selected branch",
 		UnsetUpstream:                       "Unset upstream of selected branch",
+		ViewSymmetricDifferenceToUpstream:   "View symmetric difference to upstream",
```

Comment:
I think `Compare to upstream` is easier to grok for the average reader. I only just learnt what `symmetric difference` meant.

Reply:
I also didn't know the term "symmetric difference" before, I just learnt it from reading git's documentation of the triple-dot syntax. I'm not hung up on using the term.

But I don't like "compare" because it sounds like we are showing a diff against upstream. That's also useful, but it's a different command. (And I only just realized that we have that command already in the `W` menu in the branches panel. Awesome!)

Maybe "Show divergence from upstream?"

Reply:
Yep 'Show divergence from upstream?' (what you've got in the most recent commit) works well